web-marketplace/src/components/organisms/RegistryLayout/RegistryLayout.Header.tsx

```diff
@@ -54,7 +54,9 @@ const RegistryLayoutHeader: React.FC = () => {
   const isTransparent = useMemo(() => getIsTransparent(pathname), [pathname]);
   const borderBottom = useMemo(() => getBorderBottom(pathname), [pathname]);
 
-  const { showProjects, showCreditClasses, isIssuer } = useProfileItems({});
+  const { showProjects, showCreditClasses, isIssuer } = useProfileItems({
+    address: wallet?.address,
```

Comment:
this will break how the following check works: https://github.com/regen-network/regen-web/pull/2198#discussion_r1399053857 since it assumes `address` to be set as long as we are looking at other profiles
should we rather move this in `useProfileItems` directly and try to follow the same logic as in the hooks used in `useProfileItems`? ie `const { wallet } = useWallet();
  const activeAddress = wallet?.address ?? address;`

Reply:
Make sense, I've handle that here: [c284e7e](https://github.com/regen-network/regen-web/pull/2214/commits/c284e7e5751ec219ebe8c5f77c8132057bb48304)